lib/full-site-editing/edit-site-export.php

```diff
@@ -5,6 +5,38 @@
  * @package gutenberg
  */
 
+/**
+ * Parses wp_template content and injects the current theme's
```

Comment:
Would a more accurate description here be that it removes the theme attribute, instead of stating that it injects a new one?

Reply:
Good catch. I agree.